includes/Experiments.php

```diff
@@ -365,6 +365,17 @@ public function get_experiments() {
 				'group'       => 'general',
 				'default'     => true,
 			],
+			/**
+			 * Author: @spacedmonkey
+			 * Issue: #6943
+			 * Creation date: 2021-04-07
+			 */
+			[
+				'name'        => 'videoOptimizationPublished',
+				'label'       => __( 'Video optimization pre-publish checklist', 'web-stories' ),
+				'description' => __( 'Transcode already uploaded videos.', 'web-stories' ),
```

Comment:
```suggestion
				'description' => __( 'Optimize already uploaded videos.', 'web-stories' ),
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/7130/commits/3cd9d965b0b61bbf0c021092776f2a5549420d61